airbyte-integrations/connectors/destination-redshift/src/test/java/io/airbyte/integrations/destination/redshift/RedshiftConnectionTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.redshift;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.protocol.models.v0.AirbyteConnectionStatus;
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+
+public class RedshiftConnectionTest {
+
+  private final JsonNode config = Jsons.deserialize(IOs.readFile(Path.of("secrets/config.json")));
```

Comment:
TIL integration test creds are available to unit tests 😰 

can you move this file into test-integration, since it actually hits redshift for real? (but +1 on making it not extend the base DAT class)

Reply:
ok I assume I should move `RedshiftFileBufferTest` too? both of the unit tests need a real config

Reply:
Oh derp. Yeah that would be ideal